### DISCUSSION TITLE
Some CDNs or links are not shown on cdnjs.com

### DISCUSSION THREAD

**Discussion by rajan-31** [cdnjs/cdnjs#14179] [General] [answered]
Some CDN links are not showing on cdnjs.com, for example:

https://cdnjs.com/libraries/tailwindcss

Is there any issue anybody else may have noticed?

- I tried using other browser, incognito mode, also turned off ad blocker still the same issue.
- I checked other versions from drop down list, but it shows loading and then nothing shows up.

![image](https://user-images.githubusercontent.com/62245685/212085298-08827289-a647-498e-8fab-ec4ededf8a21.png)

**Comment by MattIPv4** [comment, marked as answer]
It looks like the 2.x versions have files (https://cdnjs.com/libraries/tailwindcss/2.2.19), but the 3.x versions do not. Please open an update request in cdnjs/packages so we can look at updating the configuration used to pull in files for the library.

**Reply by rajan-31** [reply]
okay, I will do that. Thanks.

**Reply by rajan-31** [reply]
Hey, I took a look at documentation of Tailwindcss. I realized that from "tailwindcss 3.0.0" onwards it's changed how Tailwindcss works. So that's why I guess files are missing. (ref: https://tailwindcss.com/blog/tailwindcss-v3#play-cdn)

Is there any way, we could add a disclaimer for that on cdnjs website at https://cdnjs.com/libraries/tailwindcss?

**Reply by rajan-31** [reply]
@MattIPv4
